new-dti-website/components/initiatives/data/initiatives.json

```diff
@@ -15,8 +15,7 @@
       "height": 328
     },
     "description": "At the Tompkins County Public Library, our members give children in the Ithaca community the opportunity to grow and learn about other technical careers. We introduce code and design through beginner-friendly, interactive workshops with Cornell students, providing individualized feedback. Young students can learn Scratch to create their own games and animations, and Figma to make eye-catching visuals with shapes, colors, and more!",
-    "mobileDescription": "@Tompkins County Public Library, we introduced code & design with workshops that taught Figma and Sketch.",
-    "eventName": "Tompkins County Public Library"
```

Comment:
this is causing a linting error, if we not using eventName anywhere we could remove it from everything else, and also from the type

Reply:
wait ok i messed up smth but i think i fixed it here/